.ci/test/run.sh

```diff
@@ -24,7 +24,7 @@ run_tests() {
     sudo pkill -KILL -x "$name" || true
   done
 
-  sudo chown -R "${USER:-$(whoami)}" .
+  sudo chown -R "${USER:-$(whoami)}" . || true
```

Comment:
sudo does nothing on mac (it's aliased to a no-op), so tar can (and does) fail with `EPERM`.

Reply:
I mean it's disabled in CI scripts since sudo isn't needed on macOS and using it actually broke something there (probably homebrew package installation, or something like that). To keep the same scripts working everywhere, we alias sudo to nothing.

https://github.com/gsliepen/tinc/blob/1.1/.ci/test/run.sh#L55-L57

https://github.com/hg/tinc/runs/6604277651?check_suite_focus=true#step:6:12198